helpers.sh

```diff
@@ -44,9 +44,10 @@ llvm_default_version() {
 
 # $1 - toolchain name
 llvm_version() {
-  local toolchain="$1"
-  local toolchain_name="$(echo $toolchain | cut -d '-' -f 1)"
-  local toolchain_version="$(echo $toolchain | cut -d '-' -f 2)"
+  local toolchain, toolchain_name, toolchain_version
+  toolchain="$1"
+  toolchain_name="$(echo "$toolchain" | cut -d '-' -f 1)"
+  toolchain_version="$(echo "$toolchain" | cut -d '-' -f 2)"
```

Comment:
Nitpicking here, but this is standard shell variable initialization to the degree I can tell and the new version is much more foreign to 99% of people, I'll argue. We should probably just disable this warning if it complains about *that*.

Reply:
That was the warning:
https://www.shellcheck.net/wiki/SC2155